tools/cli/bin/jetpack.js

```diff
@@ -2,24 +2,52 @@
 
 import process from 'process';
 import { fileURLToPath } from 'url';
-import { checkCliLocation, compareComposerVersion } from '../helpers/checkEnvironment.js';
 
 /**
- * Checks for executing the CLI within a different monorepo checkout.
+ * Async import with better error handling.
+ *
+ * @param {string} path - Path to import from.
+ * @returns {*} - Return from `import()`.
  */
-try {
-	await checkCliLocation();
-} catch ( error ) {
-	console.error( error );
-	console.error( 'Something unexpected happened. See error above.' );
-	process.exit( 1 );
+async function guardedImport( path ) {
+	try {
+		return await import( path );
+	} catch ( error ) {
+		const identity = v => v;
+		const chalk = await import( 'chalk' ).then(
+			m => m.default,
+			() => ( {} )
+		);
+
+		if ( error.code === 'ERR_MODULE_NOT_FOUND' ) {
+			// if pnpm install hasn't been run, the import() will fail here.
+			console.error(
+				'Something is missing from your install. Please run `pnpm install` and try again.'
```

Comment:
Since at this point we're certain that the `pnpm install` should be run, let's output this text after the original error message? I know for certain that people tend to read the last line only, I know I do at least. That should get most people through the problem without having to deal with the original message.

Reply:
> I know for certain that people tend to read the last line only, I know I do at least.

So, _some_ people.